tools/assetpipelineeditor/mainwindow.cpp

```diff
@@ -337,6 +350,21 @@ void MainWindow::openFile()
     setFilePath(filePath);
 }
 
+void MainWindow::compressFile()
```

Comment:
Probably shouldn't be name compress since it'll be more generally about exporting.

Reply:
done

Reply:
done